openedx/features/smart_referral/views.py

```diff
@@ -0,0 +1,23 @@
+from edx_rest_framework_extensions.auth.session.authentication import SessionAuthenticationAllowInactiveUser
+from rest_framework.decorators import api_view, authentication_classes, permission_classes
+from rest_framework.permissions import IsAuthenticated
+from rest_framework.response import Response
+from rest_framework.status import HTTP_200_OK, HTTP_400_BAD_REQUEST
+
+from .serializers import SmartReferralSerializer
+from .tasks import task_referral_and_toolkit_emails
+
+
+@api_view(['POST'])
+@permission_classes([IsAuthenticated])
+@authentication_classes([SessionAuthenticationAllowInactiveUser, ])
+def send_initial_emails(request):
+    serializer = SmartReferralSerializer(context={'request': request}, data=request.data, many=True, allow_empty=False)
+
+    if not serializer.is_valid():
+        return Response(serializer.errors, status=HTTP_400_BAD_REQUEST)
+
+    smart_referrals = serializer.save()
+    contact_emails = [referral.contact_email for referral in smart_referrals]
+    task_referral_and_toolkit_emails(contact_emails, request.user.email)
```

Comment:
Let's please use `.send` or `.apply_async` instead of just calling this task function. If we have to use it like this then there is no point to create a task.